bloom-housing/bloom
# fix: program/preference keys length uncap

## Issue Overview

This PR addresses [#issue](https://github.com/housingbayarea/bloom/issues/606)


## Description
Currently the length of both a multiselect question title and a multiselection question's option title are uncapped, but there is a cap at the application level. This removes that cap of 128 characters

## How Can This Be Tested/Reviewed?
Go to the partner's site and create a new preference with more than 128 characters as the title, and create an option for that preference with more than 128 characters as its title 

add that new preference to a listing

go to the public site and apply to that listing, making sure to select your new preference

the data should save successfully  

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
